GitHub-Test-DHprojects/Git-Test4-FelipeB.md

```diff
@@ -0,0 +1,15 @@
+# Git-Test 4
+
+
+### Van Gogh "Tortured Genius"
+
+By Felipe Bassi
+
+The Van Gogh, *"Tortured Genius"* project goal was to understand whether Van Gogh's remarkableworks were due to his mental and personal issues or what exactly was the secret of one of the greatest artists of all time.
```

Comment:
I had never actually know the title of that painting before. But I looked it up after reading this and now I know!